docs/running-on-yarn.md

```diff
@@ -462,6 +462,14 @@ To use a custom metrics.properties for the application master and executors, upd
   <code>spark.blacklist.application.maxFailedExecutorsPerNode</code>.
   </td>
 </tr>
+<tr>
+  <td><code>spark.yarn.blacklist.initial.blacklisted.nodes</code></td>
+  <td>false</td>
+  <td>
+  Comma-separated list of strings used as initially blacklisted YARN nodes which stays always
```

Comment:
I'm wondering if we change the name/description a bit.  a few things I"m concerned with is it says blacklisted which generally has the timeout apply, the second concern is it says initial, which someone could take as meaning initially but maybe perhaps later could be.   We are adding these to the yarn blacklisting so blacklisted makes sense in that context, but perhaps we should say spark.yarn.blacklist.always.blacklisted.nodes. I always hate finding good names for configs.  I'm not fond of the 2 blacklist(ed) in the name of the config. thought about spark.yarn.scheduler.always.blacklisted.nodes?  We could go with something like spark.yarn.exclude.nodes as well since that is essentially what we are doing   @squito  thoughts?

We may want to clarify the description saying list of nodenames
The default isn't false here, so we should fix that.

Reply:
oh yeah, the default is definitely not false, thanks for catching that.

I also don't have much insight into a good name.  I don't feel super strongly but I do prefer "spark.yarn" since its yarn-specific, for now, and also prefer something with "always" etc. over"initial", as you suggested.  A bit weird having blacklisted twice, but eh, I don't really care that much.

My preferences would  be (1) "spark.yarn.exclude.nodes" (2) "spark.yarn.blacklist.always.blacklisted.nodes" (3) whatever else you want :)

Reply:
I like `spark.yarn.exclude.nodes`